src/Standards/PSR12/Docs/Files/ImportStatementStandard.xml

```diff
@@ -0,0 +1,35 @@
+<documentation title="Import Statement">
+    <standard>
+    <![CDATA[
+    Import use statements must not begin with a leading backslash.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Import statement is fully qualified and doesn't begin with a leading backslash.">
```

Comment:
Nitpick: import `use` statements are always fully qualified as that's how PHP works. That's the reason for the rule.